shared/routes/Index.html

```diff
@@ -0,0 +1,169 @@
+<div class='hero'>
+	<div class='left'>
+		<h1>Svelte</h1>
+	</div>
+
+	<div class='right'>
+		<h2>The magical disappearing UI framework</h2>
+
+		<ul>
+			<li><strong>The web's obesity crisis, solved.</strong> Svelte turns your templates into tiny, framework-less vanilla JavaScript.</li>
+			<li><strong>Simple and familiar.</strong> Build apps out of composable, easy-to-write blocks using languages you already know.</li>
+			<li><strong>Stupid fast, rock solid.</strong> Compile-time static analysis ensures the browser does no more work than it needs to.</li>
+		</ul>
+
+		<a class='learn-svelte' href='/guide'>Learn Svelte</a>
+	</div>
+</div>
+
+<style>
+	.hero {
+		position: relative;
+		padding: 2em 1em;
+		/*background-color: #333;
+		color: #f4f4f4;*/
+		background-color: white;
```

Comment:
[I defer to Emerson](http://www.goodreads.com/quotes/353571-a-foolish-consistency-is-the-hobgoblin-of-little-minds-adored) 😀